main/src/test/java/cgeo/geocaching/location/GeopointConverterTest.java

```diff
@@ -0,0 +1,108 @@
+package cgeo.geocaching.location;
+
+import cgeo.geocaching.utils.functions.Func1;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class GeopointConverterTest {
```

Comment:
I believe this is part of another PR? It was probably included here by accident. Please remove from this PR